MigrationTutorial/README.md

```diff
@@ -2,8 +2,25 @@
 This is just a temporary repo that holds the migration tutorial for Realm. This will be moved to the centralised repo for examples when one is defined.
 
 # General info
-This is a sample project to showcase how to execute migrations in Realm .NET.
-To run the application, run the command line binary like `./MigrationTutorial --schema_version [1-3]`.
+This is a sample project to showcase how to execute migrations in an application using Realm .NET.
+The application simulates a business that goes through 2 different migrations. So, `schema_version 1` just creates the database with some minimal models. `schema_version 2` and `schema_version 3` make some changes to the models, hence the application operates migrations to accomodate the changes.
+The application should only be run with the schemas in the order 1 through 3, with the ability to skip 2. If given 1 again after any run, it is assumed that the user wants to start over so the realm file will be deleted.
```

Comment:
In `V3Utils.cs` you catch an error saying that it could occur when skipping v2 (I added a comment). Here it says that skipping v2 is possible. Do those two statements contradict each other?

Reply:
They actually mean the same in the sense that with "skipping" I mean, you ran v1 and v3 but not v2. Although, if this is what you did, I still run a version of the v2 migration behind the scene but its seeding will be skipped.

Reply:
Fair enough! Looking forward to that.